### ISSUE TITLE
out.json

### ISSUE THREAD

**Issue by starlight-traveler** [Anumania/dont_starve_bot#1]
Hey, was just looking at your code for a simple Discord bot and it seems both this repository and the recent fork that is now archived has the same problem.

https://github.com/Dwigoric/dst-discord-link.git

That the out.json file and in.json file are not being populated or read from on a dedicated Don't Starve Together install, specifically a Linux based one when the json files are located in the /data/ folder.

I can fix the code myself, however I am not sure how the game reads or interacts with these files if you can share anything.

**Comment by Anumania** [comment]
i meant to publish the mod side of this for a while, i know dst's dedicated server has had issues with inconsistencies that have affected this mod before, interesting that its cropping up again, since dedicated linux definitely used to work, give me a couple hours and ill add the mod side of this to this repo.

the archived repo is not directly affiliated with me, the guy just asked for permission a while back and it seems like he doesnt want to maintain his code anymore since his repo is archived, so id assume his will remain broken forever unless you can talk to him as well.

**Comment by starlight-traveler** [comment]
Yeah it's no problem, messing around with things on the Lua plugin side I was able to get the sending message to Discord work, but not the opposite.

Dedicated server code would be appreciated, thank you! Will update this once I have something working on how I did it.

**Comment by Anumania** [comment]
ive updated the repo with the lua code, same as the one you found. there is no difference between this and the dedicated server code other than a manual check because dedicated servers return a far more precise timestamp than full clients do (for some reason). 
a fix would be appreciated if you can figure it out, feel free to fork and ill pull your changes in or just post what you fixed.
